diem-unified-charter.md

```diff
@@ -24,11 +24,9 @@ This is often a separate entity from the creator or original designer of the emb
 
 "To validate an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
 Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific validator upon request.
-Cryptographic verification can be optional as long as the emblem can be correctly interpreted.
-Therefore, to be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
+To be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
 
-
-Digital emblems can be unsigned, self-signed, or signed by a trust anchor.
+Digital emblems can be self-signed or signed by a trust anchor.
```

Comment:
```suggestion
```

Hey Rohan, this is an attempt to address your concerns while making everyone happy! By removing "unsigned" you force emblems to be signed. As Orie said, there could be different securing mechanisms. One could also think of securing mechanisms based on CT-style logs, where the authenticity comes from log inclusion (not signed!).

If we instead remove this sentence, we leave it open how emblems are secured, which is what I think the charter should do. We could simply merge the next sentence into the previous paragraph and say everything that needs to be said about securing mechanisms!

Reply:
It's a good point, saying both is the same as saying nothing.